hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/util/Utility.java

```diff
@@ -416,10 +416,19 @@ public static final long getTimestampFromFilename(String filename) {
         }
 
         date = date.replace('_', ':');
-        Instant instant = Instant.parse(date);
+        return Instant.parse(date);
+    }
+
+    public static final long getTimestampFromFilename(String filename) {
+        Instant instant = getInstantFromFilename(filename);
         return Utility.convertToNanosMax(instant.getEpochSecond(), instant.getNano());
     }
 
+    public static final String getStreamFilenameForInstant(StreamType streamType, Instant instant) {
```

Comment:
nit: to be consistent with above method
```suggestion
    public static final String getStreamFilenameFromInstant(StreamType streamType, Instant instant) {
```

Reply:
will update as suggested